api/test/integration/test_rbac_black_agent_endpoints.tavern.yaml

```diff
@@ -1586,9 +1550,6 @@ stages:
       status_code: 403
       json:
         error: 4000
-        dapi_errors:
-          master-node: # We have permission to see node name
-            error: !anystr
```

Comment:
Look at the test name. If we remove this block, this test will not do anything.  `dapi_errors` does not always show the node where the error occurred.  It depends on the `cluster:read` permission.  

We have this kind of test in other test modules as well. We should remove all of them and test it within `test_cluster_endpoints` using a distributed API call, but that is another issue. For now, do not remove it.